vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/experiments/ExperimentInformationPage.java

```diff
@@ -129,13 +143,35 @@ public void setProjectId(ProjectId projectId) {
       this.projectId = projectId;
       projectNavigationBarComponent.projectId(projectId);
       experimentListComponent.projectId(projectId);
+      itemCollection.removeAll();
+      var project = projectInformationService.find(projectId);
+      if (project.isEmpty()) {
+        return;
+      }
+      project.get().experiments().stream()
+          .map(experimentInformationService::find).filter(
+              Optional::isPresent).forEach(experiment -> {
+            ExperimentInformationPage.this.itemCollection.addExperimentItem(
+                ExperimentItem.create(experiment.get()));
+          });
+      var activeExperiment = project.get().activeExperiment();
+      itemCollection.findBy(activeExperiment).ifPresent(ExperimentItem::setAsActive);
+      enableActiveExperimentSelectionListener();
     }
 
-    public ExperimentId getActiveExperimentIdForProject() {
-      return projectInformationService.find(projectId).get().activeExperiment();
+    private void enableActiveExperimentSelectionListener() {
+      itemCollection.addClickEventListener((ComponentEventListener<ExperimentItemClickedEvent>) event -> {
+        var newActiveExperiment = event.getSource().experimentId();
+        routeToSelectedExperiment(projectId, newActiveExperiment);
+      });
+    }
```

Comment:
I'd argue that this logic is solely contained in the support area of our experimentpage and should be moved there 🤓

Reply:
there is no such thing as an support area atm, thus i needed to place it somewhere. It is subject to another PR to refactor the top level layout.